src/kdc/ndr.c

```diff
@@ -242,7 +242,7 @@ ndr_enc_delegation_info(struct pac_s4u_delegation_info *in, krb5_data *out)
 {
     krb5_error_code ret;
     size_t i;
-    struct k5buf b;
```

Comment:
`free(b.data)` could be called in the `cleanup` section before `b` is initialized.